curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-hash-table.english.md

```diff
@@ -70,18 +69,18 @@ var HashTable = function() {
 <div id='js-setup'>
 
 ```js
-  var called = 0;
-    var hash = (string) => {
-     called++;
-      var hash = 0;
-      for (var i = 0; i < string.length; i++) { hash += string.charCodeAt(i); };
-      return hash;
-    };
+var called = 0;
+var hash = string => {
+  called++;
+  var hash = 0;
```

Comment:
```js
var hash = string => {
  called++;
  var hash = 0;
```
Can we rename the local `hash` variable to something else? Like `hashed`? I don't see any reason to create a variable with the same name as the function. This might confuse beginners.

Reply:
It is a bit outside the scope of this PR, but I am fine with you committing such a change to this PR.